docs/other/performance.md

```diff
@@ -0,0 +1,11 @@
+## Improving composer build performance
```

Comment:
I'll move those sections here too. Cleans up the windows installation section which is a bit too crowded at the moment.